src/ceph-volume-zfs/ceph_volume_zfs/devices/zfs/main.py

```diff
@@ -0,0 +1,46 @@
+import argparse
+from textwrap import dedent
+from ceph_volume import terminal
+# from . import activate
```

Comment:
please remove all of these commented out imports

Reply:
@alfredodeza 
Here I would leave them in, since these are the items that need completion to be on par with ceph-volume. It is also a reminder as to what needs to be done. But if you want them out for this commit, I'll take 'm out??